webapp/app.py

```diff
@@ -100,9 +110,49 @@ def api():
     Show the static api page
```

Comment:
This is no longer static, the docstring should be updated (or removed)

Reply:
How about "Show the API reference page"?